EIPS/eip-4907.md

```diff
@@ -1,6 +1,6 @@
 ---
 eip: 4907
-title: Rental NFT, ERC-721 User And Expires Extension
+title: Rental NFT, an Extension of EIP-721
 description: Add a time-limited role with restricted permissions to ERC-721 tokens.
```

Comment:
description: Add a time-limited role with restricted permissions to EIP-721 tokens.